### ISSUE TITLE
[Pathways][US_ID] Filter dropdowns aren’t identified as clickable elements

### ISSUE THREAD

**Issue by niclui** [Recidiviz/pulse-dashboard#1902]
**What needs to be done? Why does it need to be done?**
Filter dropdowns aren’t identified as clickable elements (note the lack of yellow tag). This will affect accessibility tools.

![image.png](https://images.zenhubusercontent.com/624f7b0fe7c8d35430830c47/e82b8e6f-37dd-4f2c-8481-5df3047771f5)

**Additional context**
Add any other context about the task here.
